regions/io/core.py

```diff
@@ -221,3 +371,120 @@ def to_region(self):
 
     def _raise_error(self, msg):
         raise self.error[self.format_type](msg)
+
+    def to_crtf(self):
+        """
+        Makes shape CRTF compatible.
+        """
+        self.format_type = 'CRTF'
```

Comment:
Should this be modifiable?  I'm not sure a `Shape` should have a changeable type.  Could you explain why this is here?  I might just be missing an important use case.

Reply:
I think yes. Because, it plays a role in interconversion of file format. just makes sure that the shape object is compatible with the file format (through `_validate` function). for ex : if a region type or a coordinate frame is supported in one type and not in another(`supergalactic` frame is supported in CRTF and not in DS9 . When you try to change the format_type to the other one , it raises an error. Also, since it is restricted to only 'DS9' or 'CRTF', there is no harm?
What say?

Reply:
I generally prefer an approach where an object's state - in this case, whether it is a CRTF or ds9 region - is immutable.  Could we instead force a given object to be either CRTF or DS9, but have `to_ds9`/`to_crtf` methods?

Reply:
yes, sure. we can have.